baseboard/veyron/kernel-config

```diff
@@ -1,15 +1,15 @@
 #
```

Comment:
Uhh, is there any reason why diff of this file is THAT big?

Reply:
Due to using `make menuconfig` with kernel `5.14.0-rc6`

Reply:
I tried manually editing at first but there are some implicit config settings needed to get the Marvell WiFi working.